release/cloud_builder.sh

```diff
@@ -111,7 +111,9 @@ fi
 if [ "${BUILD_DEBIAN}" == "true" ]; then
   mkdir -p "${OUTPUT_PATH}/deb"
   cp        "${ISTIO_OUT}/istio-sidecar.deb"   "${OUTPUT_PATH}/deb/"
-  sha256sum "${ISTIO_OUT}/istio-sidecar.deb" > "${OUTPUT_PATH}/deb/istio-sidecar.deb.sha256"
+  pushd "${OUTPUT_PATH}/deb"
+    sha256sum "istio-sidecar.deb" > "istio-sidecar.deb.sha256"
+  popd
 fi
```

Comment:
i hesitate about here change. seems change code struct, need?

Reply:
previously the code would generate the following file
a6b5ab210923606b6b3fc83a102c60b2ef08270e90b7e6ce83ef13be43168c76  /workspace/go/out/linux_amd64/release/istio-sidecar.deb
now it generates the following file
a6b5ab210923606b6b3fc83a102c60b2ef08270e90b7e6ce83ef13be43168c76  istio-sidecar.deb

having the directory path is incorrect

Reply:
Not sure I follow. 

Is a6b5ab210923606b6b3fc83a102c60b2ef08270e90b7e6ce83ef13be43168c76 istio-sidecar.deb one file?

Reply:
This was the file previously generated 
https://storage.googleapis.com/istio-release/releases/1.1.0.snapshot.1/deb/istio-sidecar.deb.sha256
inside it there is one line of the form 
<sha256> <filename with path>
after the change the file has
<sha256> <filename without path>